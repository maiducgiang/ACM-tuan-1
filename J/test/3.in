4
84,61(36)
21,2(44)
38,15(43)
69,32(17)
