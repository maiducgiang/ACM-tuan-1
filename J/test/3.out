3723 44
956 45
94432 2475
137257 1980
